### PR TITLE
Add md5() in VarbinaryFunctions.java

### PR DESCRIPTION
**Pull request by jacobgao**

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestVarbinaryFunctions.java [hunk 1]
```diff
@@ -124,6 +124,13 @@ public void testFromHex()
         assertInvalidFunction("from_hex('fff')", INVALID_FUNCTION_ARGUMENT);
     }
 
+    @Test
+    public void testMd5()
+            throws Exception
+    {
+        assertFunction("md5('hashme')", VARBINARY, new SqlVarbinary(base16().decode("533F6357E0210E67D91F651BC49E1278")));
```
Comment (electrum) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestVarbinaryFunctions.java]: Add a test for empty: d41d8cd98f00b204e9800998ecf8427e

Comment (electrum) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestVarbinaryFunctions.java]: Add a helper method `sqlVarbinaryHex`:
```java
assertFunction("md5('hashme')", VARBINARY, sqlVarbinaryHex("533F6357E0210E67D91F651BC49E1278")));

File: presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java [hunk 2]
```diff
@@ -109,6 +110,14 @@ public static Slice fromHexVarchar(@SqlType(StandardTypes.VARCHAR) Slice slice)
         return Slices.wrappedBuffer(result);
     }
 
+    @Description("return the md5")
```
Comment (electrum) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java]: "compute md5 hash"

File: presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java [hunk 3]
```diff
@@ -109,6 +110,14 @@ public static Slice fromHexVarchar(@SqlType(StandardTypes.VARCHAR) Slice slice)
...
+    @ScalarFunction("md5")
+    @SqlType(StandardTypes.VARBINARY)
+    public static Slice md5(@SqlType(StandardTypes.VARCHAR) Slice slice)
```
Comment (electrum) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java]: This function should operate on `VARBINARY`. We should discuss if we want a `VARCHAR` version, and if we do, it needs to be documented as "compute md5 hash of UTF-8 encoded string"

**Comment by electrum** [comment]
This function needs to be added to the documentation

**Comment by cberner** [comment]
Your commits have the wrong email address, can you update that?

**Comment by cberner** [comment]
Merged :)
